README.md

```diff
@@ -1 +1,149 @@
+
+- LIVE PREVIEW: [Stock Exchange App](https://cancelei.github.io/Stock-Exchange-APP/#/)
+- [LIVE DEMO (2min)](https://www.loom.com/share/4f4781d918614d4896ac662d73427a9c?sid=5e421654-6ac1-4691-a5dd-813f164d3205)
+
+## 🛠 Built With:<a name="built-with"></a>
+- React
+- Redux
+- Stock API
+- HTML
+- CSS
+- JavaScript
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://html.com/">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/javascript">JavaScript</a></li>
+    <li><a href="https://react.dev/">React</a></li>
+    <li><a href="https://redux-toolkit.js.org/">Redux</a></li>
+
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- User can view a list of stocks.
+- User can view details of a stock.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+Before you proceed, ensure that you have the following installed on your machine:
+- Node.js (v12 or above)
+- npm (Node Package Manager)
+
+### Setup
+Clone this repository to your desired folder:
+
+```sh
+  git clone git@github.com:cancelei/Stock-Exchange-APP.git
+```
+
+### Install
+
+Install the project dependencies with:
+
+```sh
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  npm run build
+```
+
+### Run tests
+
+To run tests, run the following command:
+```sh
+  npm start
+```
+- Open your web browser and visit http://localhost:8080.
+- You should now see the Space Traveler's Hub application running in your browser.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Glauber Bannwart**
+
+- GitHub: [@cancelei](https://github.com/cancelei)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/gbannwart)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- User can add a stock to their favorites list.
+- User can remove a stock from their favorites list.
+- User can view their favorites list.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome! Please consider this issue link: [Project Issues](https://github.com/cancelei/Stock-Exchange-APP/issues)
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project kindly leave a star!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I am gratefull for Microverse.
```

Comment:
- Kindly  give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/) to make your project more professional.